### DISCUSSION TITLE
Snake case abolishment

### DISCUSSION THREAD

**Discussion by ChocolateMagnate** [community/community#20206] [Codespaces] [answered]
Hello there! I was coding a Python application in the codespaces (huge thanks for making such an amazing product!) In the process I made a few methods named 'viewRow' or 'viewColumn' and for them, a warning was raised saying these names ddid not comform to the snake case naming style. Although this is a convention in Python, I suggest to remove this as an error and let those people who use Pascal or camel case proceed with their preference as all of three are generally accpeted conventions. Programmers that purposefully use another naming convention still wouldn't change it, so furthermore distracting and irritating them with this warning is obsolete.
![Snake case logs](https://user-images.githubusercontent.com/95221193/177989759-d3671af4-18c1-44de-8884-b7c60d0e28a2.PNG)

**Comment by KartikM123** [comment, marked as answer]
HI @ClaraSirius! Thanks for raising this issue!

Based on those warnings, it looks like the warnings are being thrown by pylint, so I wonder if configuring your pylint to avoid snake_case warnings would solve your problem. Here's a similar problem I found on stackoverflow that may have the answer you need: https://stackoverflow.com/questions/4341746/how-do-i-disable-a-pylint-warning. 

Codespaces should respect your pylint configuration so let me know if this solution works for you! Thanks again for raising this issue!

**Reply by ChocolateMagnate** [reply]
Thanks a lot! This really helped me and all the excessive warnings are gone now. It's important to note that disabling pylint warnings in terminal didn't really work, but instead you need to find the pylint args in the settings and pass it there, then it will stop immediantely. Once again, huge thanks for your welcome assistance and unifying spirit you connect developers with!
![image](https://user-images.githubusercontent.com/95221193/178096341-2fd36a43-f799-4b79-9874-f099fc831282.png)
